converters/fromZigbee.js

```diff
@@ -1507,6 +1507,16 @@ const converters = {
             return converters.occupancy_with_timeout.convert(model, msg, publish, options, meta);
         },
     },
+    ias_zone_tamper: {
+        cluster: 'ssIasZone',
+        type: 'commandStatusChangeNotification',
+        convert: (model, msg, publish, options, meta) => {
+            const zoneStatus = msg.data.zonstatus;
+            return {
+                tamper: zoneStatus == 4 ? true : false,
```

Comment:
Can you try with: `tamper: (zoneStatus & 1<<2) > 0` instead here?

Reply:
Works finally @Koenkk

Reply:
Can we merge it now?